quodlibet/quodlibet/util/environment.py

```diff
@@ -32,6 +32,12 @@ def _dbus_name_owned(name):
         return False
 
 
+def is_gnome():
+    """If we are running under Gnome"""
+
+    return not is_plasma() and not is_unity() and not is_enlightenment()
```

Comment:
+ Checking if the user uses GNOME is not enough. Gtk.HeaderBar was introduced on Gtk 3.14. You will have to check if `Gtk.MAJOR_VERSION >= 3 && Gtk.MINOR_VERSION => 14`